docs/api_py/methods/mqvas_api.rst

```diff
@@ -5,7 +5,7 @@
 Multi-quantum variable-angle spinning
 -------------------------------------
 
-The following classes are used in simulating multi-quantum variable-angle spinning
+The following classes are used in a simulating multi-quantum variable-angle spinning
```

Comment:
@pjgrandinetti @deepanshs Should the sentence start as 

"The following classes are used in a **simulated** multi-quantum variable-angle spinning spectrum correlating..."

or

"The following classes are used when **simulating a** multi-quantum variable-angle spinning spectrum **which correlates**..."

Reply:
"The following classes are used when simulating a multi-quantum variable-angle spinning spectrum which correlates..."